src/c++/vdo/tests/OldRecoveryJournal_t1.c

```diff
@@ -178,13 +214,14 @@ static void testDirty(void)
 {
   parameters.backingFile = crashedPath;
   initializeTest(&parameters);
-  setStartStopExpectation(VDO_READ_ONLY);
-  startVDO(VDO_DIRTY);
+  readVDOFromDisk(crashedPath);
```

Comment:
Given the way it's used, 'readVDOFromDisk()" would probably be more descriptively named "saveExpectedFileContents()" or something. The current name doesn't mention that it saves the file contents elsewhere, so that's a bit of an unexpected side-effect with the current name.

Although it's not necessary, you could make "expectedData" a local variable and pass it into readVDOFromDisk() as a parameter. That would also make its usage less magical.

Reply:
Updated the function name and input parameters. Made expectedData local to testDirty().